src/index.html

```diff
@@ -1,14 +1,31 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a class="logo" href="index.html"><img src="./images/logo.png" alt="logo"></a>
```

Comment:
apply code styling [recommendations](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) from a checklist all over the page
```suggestion
      <a class="logo" href="index.html">
        <img 
          src="./images/logo.png" 
          alt="logo"
        >
      </a>
```